src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -181,6 +181,9 @@
         - name: --enable-managed-identity
           type: bool
           short-summary: (PREVIEW) Using a system assigned managed identity to manage cluster resource group.
+        - name: --aks-custom-headers
```

Comment:
"--custom-headers"?

Reply:
unfortunately you cant. `--custom-headers` seems to be a reserved keyword used by other thing. when i used that long time ago, i couldn't get the argument to pass in